packages/common/src/store/player/slice.ts

```diff
@@ -150,6 +166,11 @@ const slice = createSlice({
       const { seconds } = action.payload
       state.seek = seconds
     },
+    setPlaybackRate: (state, action: PayloadAction<SetPlaybackRatePayload>) => {
+      const { rate } = action.payload
+      state.playbackRate = rate
+      window.localStorage.setItem(PLAYBACK_RATE_LS_KEY, rate)
```

Comment:
so typically reducers are pure functions that don't have side effects. we could move this side effect to a middleware or a saga. why not do redux persist for this though?